blog/2023-08-28-pulsar-sessions-in-communityovercode-aisa-2023.md

```diff
@@ -0,0 +1,54 @@
+---
+author: tison
+title: "Recap: Apache Pulsar Sessions in CommunityOverCode Asia 2023"
+date: 2023-08-30
+---
+
+The [CommunityOverCode Asia 2023](https://apachecon.com/acasia2023/) conference (previously known as ApacheCon Asia) has been held from August 18th to August 20th.
+
+The conference gathers adopters, developers, engineers, and technologists from some of the most influential open-source communities in the world. To date, there has been a total of over 100 proposals submitted by presenters from AWS, Huawei, Tencent Cloud, StreamNative, WeBank, and many more.
+
+We are excited to have various Apache Pulsar-related sessions on this conference, including integrations with IoT technologies, applications on cloud, the Pulsar community development, and so on.
+
+<!--truncate-->
+
+## Featured sessions
+
+Let's have a quick look at some of the featured sessions about Apache Pulsar.
+
+### Apache Pulsar 3.0: First LTS release and new features
+
+Zike Yang, Software Engineer, StreamNative
+
+The Apache Pulsar community recently launched [Apache Pulsar 3.0](https://pulsar.apache.org/blog/2023/05/02/announcing-apache-pulsar-3-0/), the first LTS release of Pulsar. In this talk, we will delve into the importance of Pulsar's LTS version. We'll also cover the major features introduced in Pulsar 3.0, including a new load balancer, support for large-scale delayed messages, and Direct IO optimization.
+
+### Huawei Terminal Cloud optimizes Apache Pulsar in container scenarios
+
+Lin Lin, SDE Expert, Huawei
+
+Apache Pulsar is a cloud-native message queue that, based on its storage separation architecture, can often shrink the compute layer to save resources during periods of low traffic. We made a lot of optimizations for Apache Pulsar in containerization scenarios. Such as: At present, the Pulsar load balancing algorithm relies on the past load data of nodes, and the process of achieving balance is relatively slow. When HPA is enabled, node capacity expansion may be triggered during load balancing, and capacity expansion triggers new load balancing. How can we optimize to make Pulsar more cloud-native?
+
+### Build iot streaming data platform based on Apache StreamPipes and Apache Pulsar
+
+Zhenlu Wu, Professional teacher of the Department of Internet of Things Engineering, Guangdong Ocean University
+
+Apache StreamPipes is an Industrial iot streaming data development toolbox for non-technical users to build message transfer between pipeline components based on various message queue systems. This talk will explore how to implement the message transport layer of Apache StreamPipes using Apache Pulsar so that Apache StreamPipes can take advantage of Pulsar's advantages, including flexible scalability, high-performance message transport, low end to end latency, and more. At the same time, this talk will also introduce the practical case of this solution in the field of industrial Internet of things.
```

Comment:
```suggestion
Apache StreamPipes is an industrial IoT streaming data development toolbox for non-technical users to build message transfer between pipeline components based on various message queue systems. This talk will explore how to implement the message transport layer of Apache StreamPipes using Apache Pulsar so that Apache StreamPipes can take advantage of Pulsar's advantages, including flexible scalability, high-performance message transport, low end-to-end latency, and more. At the same time, this talk also introduces the practical case of this solution in the field of industrial Internet of Things.
```